content/millennium/dstu2/conformance.md

```diff
@@ -1,58 +1,10 @@
 ---
-title: Conformance | DSTU 2 API
+title: Conformance | DSTU2 API
```

Comment:
DSTU 2

Reply:
Made the change here: [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/7a5c7155c88812a895674b0212f9b08f67a89fa7#diff-46b7f8ac9eb5bf334f7a43b321c061afL2)